bin/post_compile

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+GETTEXT_SHA256="2f991a49b3fb616a032490c907cb700966c5fd998d2aa12b86d547a93746ed66"
+GETTEXT_TARBALL="https://assets.mofoprod.net/gettext/gettext-0.20.1.tar.xz"
```

Comment:
FYI this was compiled in a Heroku Dyno and uploaded to the mofo-assets s3 bucket for us to use here.